### PR TITLE
Add onboarding screen

### PR DESCRIPTION
**Pull request by OmarGtz**
I have added a onboarding screen

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -6,13 +6,16 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:name="com.example.mygitappexample.OnboardingFragment"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent" />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintBottom_toBottomOf="parent"
+        tools:layout="@layout/fragment_onboarding"
+        />
```
Comment (JAnt369) [inline app/src/main/res/layout/activity_main.xml:19]: Todo correcto by JAntonio

File: app/src/main/res/layout/fragment_onboarding.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
```
Comment (JAnt369) [inline app/src/main/res/layout/fragment_onboarding.xml:6]: Revisando el fragment  by JAntonio

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (JAnt369) [inline app/src/main/res/layout/activity_main.xml:19]: Ya estoy revisando el fragmento  by JAntonio

**Review by JAnt369 — APPROVED** [review]

**Comment by JAnt369** [comment]
Listo, ya quedo el Aprove

**Review by Jaaziel37 — COMMENTED** [review]
good

**Pull request merged**